### PR TITLE
Cargo: update to tokio 1.0 

### PR DESCRIPTION
**Pull request by vrutkovs**
Deps updated:

* reqwest 0.11
* dkregistry to 24af5d417b7769db63238bf4667113bda2a2bb03
* hyper to 0.14
* actix-0.11
* actix-web 4.0.0-beta.3
* actix-service 2.0.0-beta.4

TODO:
* [x] actix-cors requires update

### CODE REVIEW

**Comment by vrutkovs** [comment]
/hold

query-prometheus is incorrectly invoking blocking reqwest call

**Comment by vrutkovs** [comment]
/hold cancel

Works now \o/

/retest

**Comment by vrutkovs** [comment]
```
INFO[2021-04-12T16:09:55Z] Running step e2e-openshift-e2e-test.         
INFO[2021-04-12T16:24:35Z] Step e2e-openshift-e2e-test succeeded after 14m40.046090934s. 
...
INFO[2021-04-12T16:33:15Z] Running step e2e-gather-extra.               
INFO[2021-04-12T16:48:15Z] pod didn't start running within 15m0s: 
```

/retest

File: cincinnati/Cargo.toml [hunk 1]
```diff
@@ -5,45 +5,45 @@ authors = ["Alex Crawford <crawford@redhat.com>"]
 edition = "2018"
 
 [dependencies]
-actix-web = "^3.3.2"
+actix-web = "^4.0.0-beta.3"
```
Comment (PratikMahajan) [inline cincinnati/Cargo.toml:8]: are we sure we want to upgrade to a beta version?

Comment (vrutkovs) [reply cincinnati/Cargo.toml:8]: Yeah, beta versions are the only ones using tokio 1.0.

It appears these have been stabilized for quite time, so I'm not too worried about it

**Review by PratikMahajan — APPROVED** [review]
/lgtm

**Pull request merged**